substrate/client/network/src/discovery.rs

```diff
@@ -1320,6 +1430,10 @@ mod tests {
 											to_discover[swarm_n].remove(&other);
 										},
 										DiscoveryOut::RandomKademliaStarted => {},
+										DiscoveryOut::ClosestPeersFound(..) => {},
```

Comment:
nit: The intention behind this is to include them in metrics in a followup?

Reply:
This event is used to find provider addresses with libp2p backend, because it discards the cached addresses returned with the `FIND_PROVIDERS` response.